app/gui/view/debug_scene/component-group/src/lib.rs

```diff
@@ -70,22 +75,37 @@ struct MockEntries {
 impl MockEntries {
     fn new(count: usize) -> Rc<Self> {
         Rc::new(Self {
-            entries: PREPARED_ITEMS.iter().cycle().take(count).map(|&label| label.into()).collect(),
+            entries: PREPARED_ITEMS
+                .iter()
+                .cycle()
+                .take(count)
+                .map(|&(label, icon)| entry::Model {
+                    icon,
+                    highlighted_text: GlyphHighlightedLabelModel {
+                        label:       label.to_owned(),
+                        highlighted: if label == "convert" {
+                            vec![(Bytes(0)..Bytes(3)).into()]
```

Comment:
0 and 3 are magic. I know these are tests, but still its worth refactoring them to well named vars